src/index.js

```diff
@@ -24,5 +25,6 @@ export default {
     isMaximized,
     screenshot,
     generateThumbnail,
-    isValidDeviceName
+    isValidDeviceName,
+    watchWindowExistence
```

Comment:
`watch...Existence` is not quite good. Can we go with just `watchWindow` or is it too vague?